### PR TITLE
Bluetooth: Mesh: Fix restoring fast period divisor from settings

### PR DESCRIPTION
**Pull request by alxelax**
The Health Fast Period Divisor is stored within
the model publish parameters on the access layer.
The opposite part for divisor restoring has been missed.

Signed-off-by: Aleksandr Khromykh <aleksandr.khromykh@nordicsemi.no>

### CODE REVIEW

**Review by jhedberg — APPROVED** [review]
Looks good, but could you rephrase the commit summary to a format more consistent with the commit history:
`Bluetooth: Mesh: Fix restoring fast period divisor from settings`

**Comment by alxelax** [comment]
> Looks good, but could you rephrase the commit summary to a format more consistent with the commit history:
> `Bluetooth: Mesh: Fix restoring fast period divisor from settings`

Done for both: commit message and review summary

**Review by trond-snekvik — APPROVED** [review]
